### ISSUE TITLE
Local plugin causes a segfault

### ISSUE THREAD

**Issue by farahnur42** [faldez/tanoshi#124]
Hi, I couldn't find an answer amongst the issues and I couldn't find any documentation on where and how to set a path for the local plugin but seems like if there isn't any mentioned explicitly the application crashes with a segfault whenever the local catalogue is accessed ("illegal hardware instruction", whatever this means). Here's the [relevant log](https://gist.github.com/farahnur42/675a7261651a9a9b9fd3a2b9ee69a44c) and I hope that it's helpful.

I'm on: 
Operating System: openSUSE Tumbleweed 20200919
Kernel Version: 5.8.7-1-default
OS Type: 64-bit
Processors: 8 × Intel® Core™ i5-8250U CPU @ 1.60GHz
Memory: 7.6 GiB of RAM
Graphics Processor: Mesa DRI Intel® UHD Graphics 620

**Comment by faldez** [comment]
append this config to I've updated the Readme how to configure the path. In short add this lines to `config.yml`

```yaml
plugin_config:
  # Plugin name
  local:
    # Absolute path to manga
    path: /absolute/path/to/manga
```

mangas file structured like this
```
/absolute/path/to/manga
|_Some Series
|   |_ Volume 1.cbz
|   |_ Volume 2.cbz
|_Another Series
    |_ Volume 1.cbz
```

**Comment by farahnur42** [comment]
Well the catalogue's no longer causing a segfault (and showing my files) but opening up some files in there is, not all tho. [Log right here](https://gist.github.com/farahnur42/b3cad2e604c43c51f5171270f81ba059)

**Comment by faldez** [comment]
can I have sample of your manga? I notice some format is not working, maybe due to zip crate I use.

I do consider add vertical reader, but I want to stabilize the app first

**Comment by farahnur42** [comment]
Yeah sure, if you have a Discord account that I can DM a link to that'd be appreciated. Although I'd have to add you first, the request will come from Fahim#2690.

**Comment by farahnur42** [comment]
Yeah sure, do you have a Discord account/server that I can DM/post a (Mega) link to? It'll be coming from Fahim#2690.

**Comment by faldez** [comment]
I do have discord, it's faldez#8526

**Comment by faldez** [comment]
this should be fixed in v0.24.0
